packages/itinerary-body/src/util.ts

```diff
@@ -102,11 +102,10 @@ export function getPlaceName(
 ): string {
   // If address is provided (i.e. for carshare station, use it)
   if (place.address) return place.address.split(",")[0];
-  if (place.networks && place.vertexType === "VEHICLERENTAL") {
-    // For vehicle rental pick up, do not use the place name. Rather, use
-    // company name + vehicle type (e.g., SPIN E-scooter). Place name is often just
-    // a UUID that has no relevance to the actual vehicle. For bikeshare, however,
-    // there are often hubs or bikes that have relevant names to the user.
+  if ((place.name.match(/-/g) || []).length > 3) {
```

Comment:
Was it not possible to re-use the version of this match used elsewhere?

Reply:
I really tried to use the other function that already does this, but that one is really oriented to the station overlay and not an itinerary. It would have needed a lot of changes to make it generic enough to use both places and probably would have been a lot more code overall.